drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -371,6 +376,31 @@ static int i2c_it8xxx2_configure(const struct device *dev,
 	return 0;
 }
 
+static int i2c_it8xxx2_get_config(const struct device *dev,
+				  uint32_t *dev_config)
+{
+	struct i2c_it8xxx2_data *const data = dev->data;
+	uint32_t speed;
+
+	switch (data->bus_freq) {
```

Comment:
Return `-EIO` if `i2c_it8xxx2_configure()` has not been called to initialize `bus_freq`.

Reply:
Done.